_episodes_rmd/01-starting-with-data.Rmd

```diff
@@ -430,7 +430,7 @@ We'll learn why this is so in the next lesson.
 > > ## Solution
 > > ~~~
 > > whichPatients <- seq(2,40,2)
-> > whichDays <- c(1:5)
+> > whichDays <- 1:5
```

Comment:
I'll move this to another PR/Issue.
I can see how it goes both ways. I'm almost tempted to make a small comment that `c(1:5)` and `1:5` are the same thing.

I'll fix the conflicts and merge everything else.